homepage/homepage/components/docs/packageDocs.tsx

```diff
@@ -38,11 +38,13 @@ export async function PackageDocs({
           <section key={category.title}>
             <h2>{category.title}</h2>
             {category.children.map((child) => (
-              <RenderPackageChild
-                child={child}
-                key={child.id}
-                inPackage={packageName}
-              />
+              child.variant !== "document" && (
```

Comment:
Ability to link external documents has been added. Turning it off for now

https://typedoc.org/documents/External_Documents.html